src/DeprecatedHelpers/Search/SearchedWithNoResults.ts

```diff
@@ -6,6 +6,7 @@ export const searchedWithNoResults = (
 ): SearchedWithNoResults => {
   return {
     action: ActionType.searchedWithNoResults,
+    activePill: props.activePill,
```

Comment:
I know in [a previous review comment](https://github.com/artsy/cohesion/pull/433#discussion_r1205528162) we explored having a `PillType` and being more explicit with a property like this but to remain consistent with other surfaces, as as [I brought up in Slack](https://artsy.slack.com/archives/C9SATFLUU/p1685026012585709?thread_ts=1685026008.427369&cid=C9SATFLUU), I think we should use the existing `context_module` pattern and add any additional `[type]Tab` ContextModule properties if we need to.

So far with have the following src/Schema/Values/ContextModule.ts to use:

```
artistSeriesTab
artworksTab
auctionTab
fairTab
galleryTab
showTab
topTab
```

It looks like the only changes we need are to add `marketingCollectionTab` and `articleTab` ContextModule options and continue to use the existing `tappedNavigationTab` and `selectedItemFromSearch` events.

Reply:
+1 to Devon's comment

Reply:
Then we dont need the `activePill` field as we already have `context_module`

Reply:
Yes, exactly. I think the only changes expected of this PR at this time are new allowed values in the `ContextModule` [file here](https://github.com/artsy/cohesion/blob/41c58e113df31ca8459e014f1f1b468642fdaf0a/src/Schema/Values/ContextModule.ts#L7).